samples/src/main/java/org/corfudb/samples/graph/GraphDB.java

```diff
@@ -0,0 +1,149 @@
+package org.corfudb.samples.graph;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.SMRMap;
+
+import java.util.*;
+
+/**
+ * Created by shriyav on 5/25/17.
+ */
+
+public class GraphDB {
+    private SMRMap<UUID, Node> vertices;
+    CorfuRuntime rt;
+
+    public GraphDB(CorfuRuntime runtime) {
+        vertices = runtime.getObjectsView()
+                .build()
+                .setStreamName("A")     // stream name
+                .setType(SMRMap.class)  // object class backed by this stream
+                .open();                // instantiate the object!
+        rt = runtime;
+    }
+
+    @Override
+    public String toString() {
+        return "This graph has " + getNumNodes() + " vertices.";
+    }
+
+    public SMRMap<UUID, Node> getVertices() { return vertices; }
+
+    public Node getNode(UUID id) { return vertices.get(id); }
+
+    public void addNode(UUID uuid) {
+        vertices.put(uuid, new Node(uuid));
```

Comment:
what if a node with the same uuid already exists ?
Define exception types? (i.e. NodeAlreadyExistsException)